### ISSUE TITLE
Parsing output and saving error output to database

### ISSUE THREAD

**Issue by Eelisland** [gpac/gpac#2237]
Thanks for reporting your issue. Please make sure these boxes are checked before submitting your issue - thank you!

- [x] I looked for a similar issue and couldn't find any.
- [x] I tried with the latest version of GPAC. Installers available at http://gpac.io/downloads/gpac-nightly-builds/
- [x] I give enough information for contributors to reproduce my issue (meaningful title, github labels, platform and compiler, command-line ...). I can share files anonymously with this dropbox: https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95

Detailed guidelines: http://gpac.io/2013/07/16/how-to-file-a-bug-properly/

This is some feature suggest concerning output message from MP4Box

Error output saved to a mariadb column of type text reveal ANSI, an example error:

```
[33m[Importer] Cannot load filter for input file "/dash/3_644495.vtt:lang:fre"
[0m[31mFailure while importing media: Requested URL is not valid or cannot be found
[0m[31mError importing /dash/3_644495.vtt:lang:fre: Requested URL is not valid or cannot be found
[0m
```

Progress statistiques parsing the output of the following command to show progress:
```
/usr/bin/MP4Box -dash 4000 -frag 4000 -rap -fps 24 ...
```
which produce
```
[Dasher] No template assigned, using $File$$FS$_dash
[MPD] Generating MPD at time 2022-07-29T14:26:59.022Z(2367 segs) AS#3.1(A) done (2367 segs) seg #2344 0.29s (7.20 %) / MPD 9372.54s 99 % %
[Dasher] End of Period 
[Dasher] End of MPD (no more active streams)
```
is not possible.

The following is perfect:
```
/usr/bin/MP4Box -crypt drm.xml in.mp4 -out out.mp4
```
Output this clean one line updated
```
Encrypting:  100.00 %
```

To summarize, an option to show only errors and stats to command line would be very nice.

ffmpeg have options to do exactly that
```
ffmpeg ... -v error -stats  
```

Hope this make sense
Best,

**Comment by jeanlf** [comment]
To enable only error messages: `-logs=all@error`

Progress events in MP4Box (crypto, dash, fragmentation and spliting) are logged through tool application + level info.

So using `-logs=all@error:app@info` will only give you errors and progress.

You can use `-proglf` to force using new lines at each progress call.

For gpac, reporting is off by default but can be enabled using `-r`.

Closing as not really an issue, but feel free to continue discussion here, there's room for improvement.
